pkg/infra/mock/metadata_mock.go

```
@@ -99,19 +99,30 @@ func azureMetadataMockHandler() http.Handler {
 
-			if _, err := rw.Write([]byte("400 Bad Request")); err != nil {
+			if _, err := rw.Write([]byte("404 Not Found")); err != nil {
 				log.Fatal(err)
 			}
 
 			return
 		}
 
+		// Alternate between responding with the two event types.
+		var currentEventType string
+		switch lastAzureEventType {
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (no shared variables above switch)

(wsl_v5)